src/App.jsx

```diff
@@ -0,0 +1,27 @@
+import "./App.css";
+import Outlet from "./components/Outlet/Outlet";
+import LeftSidebar from "./components/LeftSidebar/LeftSidebar";
+import Navbar from "./components/Navbar/Navbar";
+import RightSidebar from "./components/RightSidebar/RightSidebar";
+
+function App() {
+
+  
+  return (
+    <>
+      <div className="App">
+        <div className="container1">
```

Comment:
read about css selector naming

Reply:
https://www.freecodecamp.org/news/css-selectors-cheat-sheet-for-beginners/#id I am gonna read this tomorrow 👍